src/status_im/ui/screens/browser/default_dapps.cljs

```diff
@@ -149,6 +149,10 @@
              :dapp-url    "https://3box.io/"
              :photo-path  "contacts://3box"
              :description "Create and manage your Ethereum Profile."}
+            {:name        "Aragon"
+             :dapp-url    "https://aragon.staging.aragonpm.com/#/aracon.aragonid.eth/"
```

Comment:
We will deploy to mainnet in the next few hours, the correct URL will be `https://mainnet.aragon.org`.

Reply:
the correct link was added in https://github.com/status-im/status-react/pull/7347